### PR TITLE
bumps node manager version

### PR DESCRIPTION
**Pull request by yosemitebandit**
PTAL

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+sipauthserve-public (5.1.0) unstable; urgency=medium
```
Comment (yosemitebandit) [inline debian/changelog]: what is "unstable" here and do we need it?

Comment (shaddi) [inline debian/changelog]: That's the distribution; I guess it defaults to 'unstable', one of the Debian branches. Not important for us.

**Comment by shaddi** [comment]
This is a non-API breaking bugfix change right? Let's do a micro version bump instead.

**Comment by yosemitebandit** [comment]
good point

**Comment by shaddi** [comment]
lgtm

**Pull request merged**